src/vs/workbench/services/suggest/browser/simpleCompletionModel.ts

```diff
@@ -208,10 +208,33 @@ export class SimpleCompletionModel {
 				// Then by file extension length ascending
 				score = a.fileExtLow.length - b.fileExtLow.length;
 			}
```

Comment:
Early exit for already completed ones to simplify flow of function:

```ts
			if (score !== 0) {
				return score;
			}
```

This could sit inside the `if (!isArg...)` block, then remove the `score === 0` case from the if below.